TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/bot/AndroidMovementService.java

```diff
@@ -1,146 +0,0 @@
-package com.novoda.tpbot.bot;
-
-import android.app.PendingIntent;
-import android.app.Service;
-import android.content.BroadcastReceiver;
-import android.content.Context;
-import android.content.Intent;
-import android.content.IntentFilter;
-import android.hardware.usb.UsbDevice;
-import android.hardware.usb.UsbManager;
-import android.os.IBinder;
-import android.support.annotation.Nullable;
-
-import com.novoda.notils.logger.simple.Log;
-import com.novoda.notils.logger.toast.Toaster;
-import com.novoda.support.Optional;
-
-public class AndroidMovementService extends Service implements MovementService {
```

Comment:
Replaced by the `BoundAndroidMovementService` which simply delegates to the `DeviceConnection`. We want `Services` and `Activities` to be super dumb to make testing easier and to allow switching out of components.